spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1451,6 +1451,13 @@ A client for the Spring Cloud Stream schema registry can be configured using the
   }
 ----
 
+[NOTE]
+====
+A decorator bean of type `CachingRegistryClient` is available if you want to cache responses from `fetch` operations. In your application
+just create an instance of that bean and it will replace the default version. It uses Spring caching abstraction, so any cache provider that
```

Comment:
Is this correct? We are not using a `BeanPostProcessor` to replace the original bean, so creating a new instance is more likely to create a secondary bean (maybe we should advise them to use `@Primary`). A code sample would be helpful.